File: 3120. Count the Number of Special Characters I.java
class Solution {
    public int numberOfSpecialChars(String word) {
        int[] lower = new int[26];
        int[] upper = new int[26];
        for (int i = 0; i < word.length(); i++) {
            char c = word.charAt(i);
            if (c <= 'z' && c >= 'a') {
                lower[c-'a']++;
            } else {
                upper[c-'A']++;
            }
        }
        int answer = 0;
        for (int i = 0; i < 26; i++) {
            if (lower[i] > 0 && upper[i] > 0) {
                answer++;
            }
        }
        return answer;
    }
}

File: 3120. Count the Number of Special Characters I.go
func numberOfSpecialChars(word string) int {
	lower := [26]int{}
	upper := [26]int{}
	for _, c := range word {
		if c >= 'a' && c <= 'z' {
			lower[c-'a']++
		} else if c >= 'A' && c <= 'Z' {
			upper[c-'A']++
		}
	}
	answer := 0
	for i := 0; i < 26; i++ {
		if lower[i] > 0 && upper[i] > 0 {
			answer++
		}
	}
	return answer
}